Microsoft.Toolkit.Uwp.SampleApp/SamplePages/Animations/Effects/FadeBehaviorXaml.bind

```diff
@@ -0,0 +1,43 @@
+<Page
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:interactivity="using:Microsoft.Xaml.Interactivity"
+    xmlns:interactions="using:Microsoft.Xaml.Interactions.Core"
+    xmlns:ani="using:Microsoft.Toolkit.Uwp.UI.Animations"
+    xmlns:media="using:Microsoft.Toolkit.Uwp.UI.Media"
+    xmlns:behaviors="using:Microsoft.Toolkit.Uwp.UI.Behaviors"
+    xmlns:core="using:Microsoft.Xaml.Interactions.Core"
+    mc:Ignorable="d">
+
+  <Button Background="Gray" Width="200" Height="200" HorizontalAlignment="Center" VerticalAlignment="Center">
+    <media:UIElementExtensions.VisualFactory>
+      <media:PipelineVisualFactory>
+        <media:BlurEffect x:Name="ImageBlurEffect" IsAnimatable="True"/>
+      </media:PipelineVisualFactory>
+    </media:UIElementExtensions.VisualFactory>
+    <ani:Explicit.Animations>
+      <ani:AnimationSet x:Name="BlurAnimation">
```

Comment:
Yeah not sure what happened here, but this used to be the Fade example? Looks like a mistake in e5ee018adb4602fa66c7470d33d4ab1bbe0aa8c3

Reply:
Whoops, sorry! 😅
Fixed in https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3639/commits/6196cdf3282f5028223ed02a0145f721b2ec662b, good catch!